config.go

```diff
@@ -144,15 +143,14 @@ func NewConfig() (*Config, error) {
 	}
 
 	filepath := path.Join(d, configDir, configFile)
-	file, err := openFile(filepath)
-	if err != nil {
+	b, err := os.ReadFile(filepath)
+	if err != nil && !errors.Is(err, os.ErrNotExist) {
```

Comment:
do we handle missing file case correctly? and also do we test for it?

Reply:
i think we do, i added one more test

Reply:
i've added a comment to clarify this